src/EditorFeatures/Core/DocumentationComments/CopilotGenerateDocumentationCommentProvider.cs

```diff
@@ -206,14 +207,14 @@ private static async Task<IReadOnlyList<ProposedEdit>> GetProposedEditsAsync(
 
                 if (edit.SymbolName is not null)
                 {
-                    symbolKey = edit.TagType.ToString() + "- " + edit.SymbolName;
```

Comment:
This was silly, the space is unnecessary and the JsonConvert I am doing on the Conversations side was removing the space at times.